common/changes/@itwin/core-frontend/arun-draco_2025-09-16-19-28.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-frontend",
+      "comment": "Fix loaders.gl issue with resolving draco static assets",
```

This was never released to users officially, so there is nothing thats actually being fixed
```suggestion
      "comment": "",
```